### ISSUE TITLE
Vocabulary can be clarified

### ISSUE THREAD

**Issue by cetome** [OWASP/IoT-Security-Verification-Standard-ISVS#54]
There are a few elements of vocabulary throughout the ISVS. Mainly, the ISVS is related to the verification of the security in IoT systems, which comprise devices, communications, and processing in mobile applications, in the edge, in Cloud systems.

I suggest to refine the vocabulary used in the introduction to reflect this in the **FrontIsPiece**:
1. The usage of IoT application is unclear and confusing. I would suggest to use "IoT system or systems", or "IoT asset". An application can be understood as "the software that runs IoT" or as "the application of an IoT system to perform a function".
2. When referring to "embedded applications and the IoT ecosystem", it looks like to focus are applications. I would rephrase as securing the system and its different components (firmware, communication, etc.)

**Comment by scriptingxss** [comment]
I agree. We are working to clarify these usages and plan to update content before its release in the next couple of weeks.

**Comment by cbassem** [comment]
I agree as well. The current vocabulary is confusing. How about the following. If we start from the premises that IoT is a system of systems and that the ISVS currently focusses on the IoT system (connected product) that is part of an IoT ecosystem. 
We can say that:
- We have the IoT ecosystem, i.e. the system of systems (chapter V1) 
- Some of these systems are connected products/devices/IoT systems. These are composed of different components: hardware, software platform, communication and user space applications.  
- Other systems are for example, mobile applications, web Applciations, Cloud. 

What do you think? Does this make it somehow, less confusing... ?

**Comment by cetome** [comment]
I'm happy with this definition and scope.
This is also a great way to relate to other OWASP projects (so we don't reinvent the wheel for mobile apps, Cloud, API, etc).
